repo-projects/azure-activedirectory-identitymodel-extensions-for-dotnet.proj

```diff
@@ -14,47 +14,49 @@
       <MicrosoftIdentityModelJsonWebTokensProject>$(ProjectDirectory)src/Microsoft.IdentityModel.JsonWebTokens/Microsoft.IdentityModel.JsonWebTokens.csproj</MicrosoftIdentityModelJsonWebTokensProject>
       <MicrosoftIdentityModelAbstractionsProject>$(ProjectDirectory)src/Microsoft.IdentityModel.Abstractions/Microsoft.IdentityModel.Abstractions.csproj</MicrosoftIdentityModelAbstractionsProject>
       <MicrosoftIdentityModelLoggingProject>$(ProjectDirectory)src/Microsoft.IdentityModel.Logging/Microsoft.IdentityModel.Logging.csproj</MicrosoftIdentityModelLoggingProject>
+      <MicrosoftIdentityModelProtocolsOpenIdConnectProject>$(ProjectDirectory)src/Microsoft.IdentityModel.Protocols.OpenIdConnect/Microsoft.IdentityModel.Protocols.OpenIdConnect.csproj</MicrosoftIdentityModelProtocolsOpenIdConnectProject>
```

Comment:
Why do we need to build these in source-build? I thought these dependencies aren't needed in source-build - we only ship the "user-jwts" CLI tool inbox in .NET, and not the JwtBearer, WSFederation packages.